runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -11,9 +11,7 @@
 import lombok.extern.slf4j.Slf4j;
 
 import org.corfudb.protocols.logprotocol.StreamCOWEntry;
-import org.corfudb.protocols.wireprotocol.TokenResponse;
-import org.corfudb.protocols.wireprotocol.TokenType;
-import org.corfudb.protocols.wireprotocol.TxResolutionInfo;
+import org.corfudb.protocols.wireprotocol.*;
```

Comment:
Sneaky intellij "smart import" here. There is an option when you hover on the line to import all (and only) the used apis.